src/orchestrator.ts

```diff
@@ -308,16 +309,22 @@ export class Orchestrator {
         );
     }
 
-    private callHttp(state: HistoryEvent[], method: string, uri: string, content?: string | object) {
+    private callHttp(
+        state: HistoryEvent[],
+        method: string,
+        uri: string,
+        content?: string | object,
+        headers? : { [key: string]: string },
+        tokenSource?: TokenSource) {
         if (content && typeof content !== "string") {
             content = JSON.stringify(content);
         }
 
-        const req = new DurableHttpRequest(method, uri, content as string);
+        const req = new DurableHttpRequest(method, uri, content as string, headers, tokenSource);
         const newAction = new CallHttpAction(req);
 
         // callHttp is internally implemented as a well-known activity function
-        const httpScheduled = this.findTaskScheduled(state, "Durable:Http:Async:Activity:Function");
+        const httpScheduled = this.findTaskScheduled(state, "BuiltIn::HttpActivity");
```

Comment:
Did we change this in the extension as well?

Reply:
Yes, it's part of this PR: https://github.com/Azure/azure-functions-durable-extension/pull/953